Last check before freezing chapter 5 (#93)
SquareBracketAssociates/BuildingApplicationWithSpec2: Chapters/InANutshell/InANutshell.md

@cha_core

Spec is Pharo's user interface framework. It provides the building blocks for constructing UIs, from simple windows to complex tools like browsers and debuggers. With Spec, developers can capture the layout and the interactions between the elements that compose a UI. For example, with Spec developer can express that a tool has two elements: a list on the left and an element displaying information on the left and that clicking on an item list will display detailed information about the selected item.

> In addition, Spec supports the reuse of the UI interaction logic. 

Reviewer: ```suggestion
Spec is Pharo's user interface framework. It provides the building blocks for constructing UIs, from simple windows to complex tools like browsers and debuggers. With Spec, developers can capture the layout and the interactions between the elements that compose a UI. For example, a developer can express that a tool has two components: a list on the left and a component displaying information on the right. Clicking on an item in the list will display detailed information about the selected item. In addition, Spec supports the reuse of the UI interaction logic.

```

### Application

A Spec application (an instance of the `SpApplication` class hierarchy) handles your application initialization, configuration, and resources. `SpApplication` is not a presenter because it does not have a graphical representation. An instance of `SpApplication` defines your application (keeping the backend, theme, icons, and other graphical resources), and keeps the opened windows that belong to the application, but it is not shown itself.

A Spec application also provides a way to access windows or resources such as icons, and provides abstractions for interactions with the user (inform, error, file, or directory selection).

> Finally an application provides the style used by Spec to style UI elements.  A default style is available but you can customize it as shown in Chapter *@cha_style@*.

Reviewer: ```suggestion
Finally, an application provides the style used by Spec to style UI elements.  A default style is available, but you can customize it as shown in Chapter *@cha_style@*.
```

### Application configuration

In the application initialization, you can configure the backend you want to use: Morphic (default) or GTK.

> In the future, Spec will also support Toplo (a new widget library built on top of Bloc the replacement of Morphic).

Reviewer: ```suggestion
In the future, Spec will also support Toplo, a new widget library built on top of Bloc. It will replace Morphic.
```

I am not sure about this suggestion, as I am not sure what you meant.
